ee/tabby-ui/.eslintrc.json

```diff
@@ -8,6 +8,7 @@
   ],
   "plugins": [
     "tailwindcss",
+    "@typescript-eslint",
```

Comment:
is this still required? consider remove if it's not longer necessary

Reply:
Without this plugin, the `unused-imports/no-unused-imports` will stop working when we using `Enum`